### PR TITLE
Remove references to Clinepidb in site text

### PR DESCRIPTION
**Pull request by chowington**
Fixes https://github.com/VEuPathDB/web-eda/issues/1266

### CODE REVIEW

File: src/lib/workspace/sharing/ConfirmPublicAnalysis.tsx [hunk 1]
```diff
@@ -72,7 +72,7 @@ export default function ConfirmPublicAnalysis({
                 target="_blank"
                 style={{ color: theme?.palette.primary.hue[600] }}
               >
-                ClinEpiDB Public analyses page.
+                Public analyses page.
```
Comment (bobular) [inline src/lib/workspace/sharing/ConfirmPublicAnalysis.tsx:75]: I know the original has a period, but this isn't a sentence and likely doesn't need one.

How does it look on the webpage (could you provide a screenshot pls - I am lazy!)

**Review by bobular — DISMISSED** [review]
In lieu of Ann, I did this gruelling review instead.  Had one comment/question!

File: src/lib/workspace/sharing/ConfirmPublicAnalysis.tsx (hunk 1 above)
Comment (chowington) [reply src/lib/workspace/sharing/ConfirmPublicAnalysis.tsx:75]: Hi Bob! It is a sentence actually---the sentence says "This analysis will appear on the Public analyses page." Here's the modal:

image

**Review by dmfalke — COMMENTED** [review]
I would ask for feedback on the verbiage related to the original variable name. I think it's okay, but they might prefer including something about the data being "integrated".

File: src/lib/workspace/Variable.tsx [hunk 2]
```diff
@@ -84,8 +84,7 @@ export function VariableDetails(props: Props) {
               {showMore && providerLabelLeftover}
               &nbsp;
               <HelpIcon>
-                The name of this variable in the data files that were integrated
-                into ClinEpiDB
+                The name of this variable in the orignal data files
```
Comment (chowington) [inline src/lib/workspace/Variable.tsx:87]: @dmfalke That makes sense. @danicahelb or @SheenaTomko, any feedback on the above change?

Comment (SheenaTomko) [reply src/lib/workspace/Variable.tsx:87]: I'm ok with it. @danicahelb? I know the files we integrate aren't always "original". If you want something like "The name of this variable in the data file it was integrated from" I would be ok with that too

**Review by asizemore — APPROVED** [review]
The changes look good to me! But I agree with the other comments that suggest @SheenaTomko and @danicahelb are probably the ones who need to approve the verbiage

File: src/lib/workspace/Variable.tsx (hunk 2 above)
Comment (chowington) [reply src/lib/workspace/Variable.tsx]: @danicahelb To be clear, which verbiage did you approve? Mine above or Sheena's below mine?

Comment (chowington) [reply src/lib/workspace/Variable.tsx]: I believe in the meeting she said the suggestion I put forth above. I'll merge, but feel free to correct me, @danicahelb!

**Comment by chowington** [comment]
@bobular Seems like you specifically need to approve my changes since you submitted a change request?

**Review by dmfalke — APPROVED** [review]
@chowington I could be wrong, but I think the reason you can't merge is that the branch is out of date with `main`. Can you try merging the latest and see if you're able to merge this PR?

**Comment by chowington** [comment]
@dmfalke I don't think so! I just merge in main and here's what I'm seeing (you might be seeing it too):

image

**Comment by dmfalke** [comment]
> @dmfalke I don't think so! I just merged in main and here's what I'm seeing (you might be seeing it too):
> 
> image

Ah, ok. I saw a different message yesterday. Let me see what I can do.

**Comment by dmfalke** [comment]
I "dismissed" @bobular's review, so you should be able to merge now.

**Pull request merged**